beers/test/factories.py

```diff
@@ -0,0 +1,19 @@
+import factory
+
+from beers.models import BeerStyle
+
+class BeerStyleFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = BeerStyle
+
+    name = 'Super Light Beer'
+    revision = '2015'
+    bjcp_class = 'Beer'
+    category = '42'
+    subcategory = 'A'
+
+    category_name = 'Super Light'
+    category_notes = 'Beers for all day football watching'
+
+    ibu_low = 2.2
```

Comment:
Use `factory.fuzzy.FuzzyDecimal(0, 15, precision=1)` here so we get random IBUs

Reply:
Also, IBU is an integer. Did you mean ABV?

Reply:
yes